process/block/metablock.go

```diff
@@ -396,6 +397,11 @@ func (mp *metaProcessor) ProcessBlock(
 		return err
 	}
 
+	err = mp.handleBlockProcessingCutoff(header)
```

Comment:
in order to work, we need 2 functions: one for block processing that will error and retry and the other separate function for endless wait **at the end of the CommitBlock function** to have a consistent data saved on disk/outport drivers.
So this function can end directly by calling:
```GO
return mp.handleBlockProcessingCutoff(header)
```

Also on shard block